Projekty/Projekt1/Deregowski_Sebastian_Janus_Dawid/Deregowski_Janus_Projekt1_KM2.ipynb

```diff
@@ -0,0 +1,774 @@
+{
```

Comment:
<p>Fragmenty kodu się powtarzają - warto byłoby być może opakować je w funkcję.</p><p> </p><p>W przypadku zadania klasyfikacji, nie stosujemy RMSE jako miary jakości (to dość poważny błąd metodologiczny)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>